pom.xml

```diff
@@ -6,7 +6,7 @@
     <name>Cassandra Reaper</name>
     <groupId>com.spotify</groupId>
     <artifactId>cassandra-reaper</artifactId>
-    <version>0.5.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
for a commercial product one might want to blow-ones-whistle, but open source projects have traditionally used the 1.0 marker around maturity and a commitment to compatibility (and not to do with a feature list, no directly per se).

let's just bump it to 0.6-SNAPSHOT ?
(the minor.minor number is meaningless in any sub-1.0 release)